databricks/koalas/plot.py

```diff
@@ -1183,12 +1257,16 @@ def pie(self, **kwds):
 
         Examples
         --------
-        >>> df = ks.DataFrame({'mass': [0.330, 4.87 , 5.97],
-        ...                    'radius': [2439.7, 6051.8, 6378.1]},
-        ...                   index=['Mercury', 'Venus', 'Earth'])
-        >>> plot = df.mass.plot.pie(figsize=(5, 5))
 
-        >>> plot = df.mass.plot.pie(subplots=True, figsize=(6, 3))
+        .. plot::
+            :context: close-figs
+
+            >>> df = ks.DataFrame({'mass': [0.330, 4.87, 5.97],
+            ...                    'radius': [2439.7, 6051.8, 6378.1]},
+            ...                   index=['Mercury', 'Venus', 'Earth'])
+            >>> plot = df.mass.plot.pie(figsize=(5, 5))
+
+            >>> plot = df.mass.plot.pie(subplots=True, figsize=(6, 3))
```

Comment:
Do we need `.. plot::` here as well? Seems like the documents are properly generated, though.

Reply:
@ueshin is correct. pandas put `.. plot::` on each plot.

https://github.com/pandas-dev/pandas/blob/v0.25.1/pandas/plotting/_core.py#L1310-L1363

Reply:
@ueshin @harupy Thanks for comment! i'll add them for all proper location. :)

Reply:
This was resolved at #865.